dash_table/Format.py

```diff
@@ -0,0 +1,286 @@
+import collections
+import inspect
+import sys
+
+
+def get_named_tuple(name, dict):
+    return collections.namedtuple(name, dict.keys())(*dict.values())
+
+
+Align = get_named_tuple('align', {
+    'default': '',
+    'left': '<',
+    'right': '>',
+    'center': '^',
+    "right_sign": '='
+})
+
+Group = get_named_tuple('group', {
+    'no': '',
+    'yes': ','
+})
+
+Padding = get_named_tuple('padding', {
+    'no': '',
+    'yes': '0'
+})
+
+Prefix = get_named_tuple('prefix', {
+    'yocto': 10**-24,
+    'zepto': 10**-21,
+    'atto': 10**-18,
+    'femto': 10**-15,
+    'pico': 10**-12,
+    'nano': 10**-9,
+    'micro': 10**-6,
+    'milli': 10**-3,
+    'none': None,
+    'kilo': 10**3,
+    'mega': 10**6,
+    'giga': 10**9,
+    'tera': 10**12,
+    'peta': 10**15,
+    'exa': 10**18,
+    'zetta': 10**21,
+    'yotta': 10**24
+})
+
+Scheme = get_named_tuple('scheme', {
+    'default': '',
+    'decimal': 'r',
+    'decimal_integer': 'd',
+    'decimal_or_exponent': 'g',
+    'decimal_si_prefix': 's',
+    'exponent': 'e',
+    'fixed': 'f',
+    'percentage': '%',
+    'percentage_rounded': 'p',
+    'binary': 'b',
+    'octal': 'o',
+    'lower_case_hex': 'x',
+    'upper_case_hex': 'X',
+    'unicode': 'c'
+})
+
+Sign = get_named_tuple('sign', {
+    'default': '',
+    'negative': '-',
+    'positive': '+',
+    'parantheses': '(',
+    'space': ' '
+})
+
+Symbol = get_named_tuple('symbol', {
+    'no': '',
+    'yes': '$',
+    'binary': '#b',
+    'octal': '#o',
+    'hex': '#x'
+})
+
+Trim = get_named_tuple('trim', {
+    'no': '',
+    'yes': '~'
+})
+
+
+def is_string_py2(value):
+    return isinstance(value, str) or isinstance(value, unicode)
+
+
+def is_string_py3(value):
+    return isinstance(value, str)
+
+
+def is_string(value):
+    return is_string_py3(value) if sys.version_info >= (3, 0) else is_string_py2(value)
```

Comment:
dash-docs was having issues in py27 because unicode is a thing.. choosing the function based on the situation and disabling F821 in package.json (py3 doesn't like seeing unicode lying around!)

Reply:
I generally think it's cleaner to do feature detection than switch based on the version number. This one has some nice options https://stackoverflow.com/questions/11301138

This one is particularly terse: `isinstance(x, ("".__class__, u"".__class__))`

Reply:
Sure. Will actually go with isinstance(x, (str, u''.__class__)) as there's no edge case for string